### PR TITLE
build: fix warning MSB4011 (props cannot be imported again. It was already imported)

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0">
 
   <Import Project="$(SolutionDir)\crosstargeting_override.props" Condition="exists('$(SolutionDir)\crosstargeting_override.props')" />
-
+  <Import Project="uno.mobile-cross-targeting-disable.props" />
```
Comment (Youssef1313) [inline src/Directory.Build.props:4]:
```suggestion

```

File: src/Uno.CrossTargetting.props [hunk 2]
```diff
@@ -23,8 +23,6 @@
 	<NoWarn>$(NoWarn);0105</NoWarn>
   </PropertyGroup>
```
Comment (Youssef1313) [inline src/Uno.CrossTargetting.props:25]:
```suggestion

<Import Project="uno.mobile-cross-targeting-disable.props" />

```

File: src/Uno.CrossTargetting.props [hunk 3]
```diff
@@ -23,8 +23,8 @@
...
 
-	<Import Project="uno.mobile-cross-targeting-disable.props" />
```
Comment (jeromelaban) [inline src/Uno.CrossTargetting.props:26]: Maybe using a condition would be enough (using a property defined in `uno.mobile-cross-targeting-disable.props`)

Comment (jeromelaban) [reply src/Uno.CrossTargetting.props:26]: We're going to remove this once we have net7 fully included (in 17.4)

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**